tools/docker/Dockerfile.build

```diff
@@ -1,4 +1,4 @@
-ARG ALPINE_BASE_IMAGE="alpine:3.18"
+ARG ALPINE_BASE_IMAGE="alpine:3.19"
```

Comment:
The image used in gitlab is 3.19.3.
https://github.com/DataDog/libddprof-build/blob/main/.gitlab-ci.yml#L10
Shouldn't use the same version here too ?

Reply:
Updated to use 3.19.3 ✅ 

https://github.com/DataDog/libdatadog/blob/5d181a07cec2fc66c9489d04c764f5cf2faac93e/tools/docker/Dockerfile.build#L1